File: server/src/main/kotlin/com/landonpatmore/models/messaging/streamable/StreamablePlayerCarInfo.kt
package com.landonpatmore.models.messaging.streamable

data class StreamablePlayerCarInfo(
    val brakeInput: Float,
    val absActivated: Boolean,
    val throttleInput: Float,
    val rpm: Int,
    val speed: Float,
    val gear: Int,
    val fuelLevel: Float,
    val fuelPercentage: Float
)

File: logger/models/streamable/StreamablePlayerCarInfo.py
from dataclasses import dataclass


@dataclass
class StreamablePlayerCarInfo:
    brakeInput: float
    absActivated: bool
    throttleInput: float
    rpm: int
    speed: float
    gear: int
    fuelLevel: float
    fuelPercentage: float
    fuelUsePerHour: float
    carsInProximity: int
